lib/pbench/common/selinux.py

```diff
@@ -0,0 +1,36 @@
+"""SELinux wrapper module.
+
+There is no "pip" installable `selinux` python module, just a shim, of sorts [1],
+which provides a way to add the installed system's `selinux` module to a
+virtualenv.  But that does not work when the virtualenv uses a version of
+python different from the python version where the real `selinux` module is
+installed.
+
+This wrapper acts as a pass-through to the real `selinux` module for the methods
+we need, when that module is present.  When that module is not present, we
+create a class to mimic the behaviors we need.  That created class declares that
```

Comment:
"[...]  When that module is not present, we provide stub routines which declare that [...]"